src/device-registry/models/KnowYourAirLesson.js

```diff
@@ -97,7 +97,7 @@ knowYourAirLessonSchema.statics = {
       };
     }
   },
-  async list({ skip = 0, limit = 1000, filter = {} } = {}) {
+  async list({ skip = 0, limit = 1000, filter = {}, user_id } = {}) {
```

Comment:
Hi @BenjaminSsempala , isn't this `user_id` already part of the `filter` generated from `utils/generate-filter`?

Reply:
@Baalmart No, For the Kya Lesson, the `user_id` not part of the filter.

Reply:
Alright @BenjaminSsempala